### PR TITLE
IP 1

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

**Comment by maijapapaya** [comment]
So starting at line 1519, some of the questions aren't labeled with numbers (and some are). I wasn't sure if you wanted me to leave them like that or add them in. I don't want to take to much liberty in changing these originals!! 😄

File: unabridged-works-volume-4/original/01-further-testimony-to-truth.adoc [hunk 1]
```diff
@@ -1157,17 +1218,17 @@ Where the light is arisen, there it is day;
...
 [.discourse-part]
 Answer.
 By waiting to feel somewhat of the life of God arising in him,
 by turning and hearkening to his witness.
 That will faithfully discover how it is with him, and where he is.
 
-Question 6. How may a man come out of the darkness of the night,
-into the light and brightness of the day?
+[.discourse-part]
+__Question 6. How may a man come out of the darkness of the night,
```
Comment (jaredh159) [inline unabridged-works-volume-4/original/01-further-testimony-to-truth.adoc:1230]: The italics don't look exactly right to me here. Did you mean to have it be:

```ASCIIDOC
_Question 6._ How may a man come out of the darkness of the night,
```

File: unabridged-works-volume-4/original/01-further-testimony-to-truth.adoc [hunk 2]
```diff
@@ -1422,7 +1505,8 @@ creating in thee) in the way of his quickening and renewing life.
 [.inline]
 === Some _Questions_ answered concerning the Spirit of _Christ,_ and the spirit of the Scribes and Pharisees
 
-Question 1. WHAT kind of spirit was the spirit of the Scribes and Pharisees?
+[.discourse-part]
+_Question 1._ WHAT kind of spirit was the spirit of the Scribes and Pharisees?
```
Comment (jaredh159) [inline unabridged-works-volume-4/original/01-further-testimony-to-truth.adoc:1509]: can you downcase the `WHAT` here?

Comment (jaredh159) [reply unabridged-works-volume-4/original/01-further-testimony-to-truth.adoc:1509]: Also, below, on line 1519, can you change it to this:

```ASCIIDOC
[.discourse-part]
_Question 2._ What kind of Spirit was the Spirit of Christ?
```

Comment (jaredh159) [reply unabridged-works-volume-4/original/01-further-testimony-to-truth.adoc:1509]: And also do the same with line 1551, move the 5 up to the line before, so it reads:

```ASCIIDOC
[.discourse-part]
_Question 5._ What did Christ judge of the Scribes and Pharisees according
```

File: unabridged-works-volume-4/original/01-further-testimony-to-truth.adoc (hunk 1 above)
Comment (maijapapaya) [reply unabridged-works-volume-4/original/01-further-testimony-to-truth.adoc:1230]: Yes, idk why that happened!

**Comment by jaredh159** [comment]
Looks good, merged.

**Pull request merged**